content/data/friends-websites.yml

```diff
@@ -48,3 +48,7 @@
 - name: Thang Chung - GitHub Repository
   url: https://github.com/thangchung
   about: Codespace by cloud-native developer and architect guy from Vietnam
+
+- name: .NET Bangkok Meetup
+  url: https://www.meetup.com/dotnet-bkk
+  about: Montly meetup about .NET in Bangkok.
```

Comment:
@kidchenko Mannnnnn proud of you.
However, a little typo `Montly => Monthly`.
After you have fixed this, I will merge your PR.

Thanks.

Reply:
Thanks @aaronamm fixing today

Reply:
cc @aaronamm done